<ui version="4.0" >
 <class>LiveClass</class>
 <widget class="QWidget" name="LiveClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Live</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="maximumSize" >
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <family>Book Antiqua</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton" >
          <property name="text" >
           <string>Scan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2" >
          <property name="text" >
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget" >
        <column>
         <property name="text" >
          <string>Channel List</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4" >
          <property name="icon" >
           <iconset resource="cinecastui.qrc" >
            <normaloff>:/CineCastUi/Resources/play.png</normaloff>:/CineCastUi/Resources/play.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3" >
          <property name="icon" >
           <iconset resource="cinecastui.qrc" >
            <normaloff>:/CineCastUi/Resources/stop.png</normaloff>:/CineCastUi/Resources/stop.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="font" >
      <font>
       <family>Book Antiqua</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Live Mode Select:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton" >
          <property name="text" >
           <string>Direct Decode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2" >
          <property name="text" >
           <string>IP Broadcast</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3" >
        <property name="title" >
         <string>Direct Decode</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" >
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" >
           <item>
            <widget class="QWidget" native="1" name="widget_2" >
             <layout class="QHBoxLayout" name="horizontalLayout_12" >
              <item>
               <widget class="QLabel" name="label_3" >
                <property name="minimumSize" >
                 <size>
                  <width>147</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>Video Decoder: </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_6" >
                <property name="minimumSize" >
                 <size>
                  <width>105</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>Auto</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_4" >
                <property name="minimumSize" >
                 <size>
                  <width>105</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>MPEG2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_3" >
                <property name="minimumSize" >
                 <size>
                  <width>105</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>H.264</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_5" >
                <property name="minimumSize" >
                 <size>
                  <width>105</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>H.265</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_7" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" >
           <item>
            <widget class="QWidget" native="1" name="widget_3" >
             <layout class="QHBoxLayout" name="horizontalLayout_9" >
              <item>
               <widget class="QLabel" name="label_5" >
                <property name="minimumSize" >
                 <size>
                  <width>147</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>Audio Output:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_2" >
                <property name="minimumSize" >
                 <size>
                  <width>105</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>SPDIF</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox" >
                <property name="minimumSize" >
                 <size>
                  <width>105</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>Analog</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6" >
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" >
           <item>
            <widget class="QWidget" native="1" name="widget" >
             <layout class="QHBoxLayout" name="horizontalLayout_11" >
              <item>
               <widget class="QLabel" name="label_6" >
                <property name="minimumSize" >
                 <size>
                  <width>147</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>Audio Decoder:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_10" >
                <property name="minimumSize" >
                 <size>
                  <width>105</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>Auto</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_11" >
                <property name="minimumSize" >
                 <size>
                  <width>105</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>MP3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_9" >
                <property name="minimumSize" >
                 <size>
                  <width>105</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>PCM</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_7" >
                <property name="minimumSize" >
                 <size>
                  <width>105</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>DTS</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_8" >
                <property name="minimumSize" >
                 <size>
                  <width>105</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text" >
                 <string>AC3</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_9" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4" >
        <property name="title" >
         <string>IP Broadcast</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4" >
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" >
           <item>
            <widget class="QLabel" name="label_7" >
             <property name="text" >
              <string>Boardcast IP:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit" />
           </item>
           <item>
            <widget class="QLabel" name="label_8" >
             <property name="text" >
              <string>Port:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2" />
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="cinecastui.qrc" />
 </resources>
 <connections/>
</ui>
